packages/pigment-css-unplugin/src/index.ts

```diff
@@ -284,6 +283,19 @@ export const plugin = createUnplugin<PigmentOptions, true>((options) => {
         }
 
         let { cssText } = result;
+
+        const slug = slugify(cssText);
```

Comment:
Can you move this at the end of the `if (isNext)` check at line https://github.com/mui/pigment-css/blob/master/packages/pigment-css-unplugin/src/index.ts#L301

since we mainly want to do this for next.js and not webpack.